libs/components/data-manager/src/lib/modules/data-manager/models/data-manager-config.ts

```diff
@@ -18,6 +18,11 @@ export interface SkyDataManagerConfig {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   filterModalComponent?: any;
+  /**
+   * A descriptor for the items that the data manager is managing. Use a plural term. The descriptor helps set the data manager's `aria-label` attributes for multiselect toolbars, search inputs, sort buttons, and filter buttons to provide text equivalents for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility).
```

Comment:
```suggestion
   * A descriptor for the data that the data manager manipulates. Use a plural term. The descriptor helps set the data manager's `aria-label` attributes for multiselect toolbars, search inputs, sort buttons, and filter buttons to provide text equivalents for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility).
```
Just my 2 cents, but I think "data" (or "data set") makes more sense here than "items," and I'd stick with "manipulates" instead from the corresponding description in the tool bar docs. If you think disagree on the verb change and think we need to stick with "manage" over "manipulate," I'd change "is managing" to "manages."

Reply:
Done